Shared/Actions/Action.gd

```diff
@@ -18,3 +18,9 @@ var trigger_times: Array = []
 signal ammunition_changed
 signal action_triggered
 signal action_released
+
+func _disable_signal_warnings() -> void:
```

Comment:
I think that instead of these functions, explicitly ignoring the warnings would be more clear. It works like this:

```gdscript
#warning-ignore:unused_signal
signal ammunition_changed
```

Reply:
Fixed.